### ISSUE TITLE
Disruptive newline in output of Squiz.WhiteSpace.SemicolonSpacing

### ISSUE THREAD

**Issue by ahtux** [squizlabs/PHP_CodeSniffer#2650]
The following example shows, that there is a disruptive newline in the output of the sniff. It would be nice, if the output looks like the output line of the ControlSignature sniff with an escaped newline.

Tested code (test.php):
```php
<php?
if (1 == 1){
    ;
}
```
The rule (test.xml):
```xml
<?xml version="1.0" encoding="UTF-8"?>
<ruleset name="PHPCS rule set">
    <rule ref="Squiz.WhiteSpace.SemicolonSpacing" />
    <rule ref="PEAR.ControlStructures.ControlSignature" />
</ruleset>
```
The command:
```sh
phpcs --report=emacs --standard="test.xml" test.php
```
What I get:
```
test.php:2:1: error - Expected "if (...) {\n"; found "if (...){\n"
test.php:3:5: error - Space found before semicolon; expected "{;" but found "{
    ;"
```
What I expect:
```
test.php:2:1: error - Expected "if (...) {\n"; found "if (...){\n"
test.php:3:5: error - Space found before semicolon; expected "{;" but found "{\n    ;"
```

**Comment by gsherwood** [comment]
I've changed the error message itself to fix this. I considered changing the various report classes, but I think that on-screen messages shouldn't be rendering newlines either - it makes the message hard to read.

Thanks for the report.
